local-setup/nginx/sites-enabled/aether.conf

```diff
@@ -138,15 +138,4 @@ server {
 
     include /etc/nginx/uwsgi_params;
   }
-
-    # ----------------------------------------------------------------------------
-    # http://aether.local/exm/ (exm)
-    location /exm/ {
```

Comment:
This was included to allow access to the health check url required by devops, has that changed?

Reply:
there is no health endpoint... :woman_shrugging:

Reply:
Seems it was removed.